### PR TITLE
fix: update classrooom services - jupyterhub in particular

### PR DESCRIPTION
**Pull request by paulstey**
This PR brings the `Classroom` services section up-to-date with the current website. Only the JupyterHub section had experienced drift, which this PR fixes.

### CODE REVIEW

File: services/classroom/jupyterhub.md [hunk 1]
```diff
@@ -5,7 +5,13 @@ icon: code-greater-than-or-equal
...
+Brown's JupyterHub service  provides a convenient, cloud-hosted way to serve Jupyter Notebooks for multiple users. Notebooks are launched within a pre-configured computing environment; users do not need to install any software packages. This set-up free environment is ideal for courses and workshops where instructors intend for students to begin coding with minimal obstacles. Jupyter’s flexibility allows instructors to pick the preferred language for a particular context, including Python, Julia, R and many more. Regardless of the language chosen, the Jupyter interface remains the same. To learn more about using Jupyter Notebooks for teaching and learning, [please visit this guide.](https://jupyter4edu.github.io/jupyter-edu-book/jupyter.html)
```
Comment (mcmcgrath13) [inline services/classroom/jupyterhub.md:8]:
```suggestion
Brown's JupyterHub service  provides a convenient, cloud-hosted way to serve Jupyter Notebooks for multiple users. Notebooks are launched within a pre-configured computing environment; users do not need to install any software packages. This set-up free environment is ideal for courses and workshops where instructors intend for students to begin coding with minimal obstacles. Jupyter’s flexibility allows instructors to pick the preferred language for a particular context, including Python, Julia, R and many more. Regardless of the language chosen, the Jupyter interface remains the same. To learn more about using Jupyter Notebooks for teaching and learning, [please visit the guide.](https://jupyter4edu.github.io/jupyter-edu-book/jupyter.html)
```

File: services/classroom/jupyterhub.md [hunk 2]
```diff
@@ -5,7 +5,13 @@ icon: code-greater-than-or-equal
...
+
+On Brown's JupyterHub, each user is provided a persistent working directory and compute resource allocation. This means the environment you are provided is only accessible by yourself and CIS support staff. 
+
+Once connected to JupyterHub servers, users enter an isolated workspace to write and run code. There are no time limit restrictions or specified lockout times, so feel free to use your personal JupyterHub server as needed and adhere to CCV's Computing Policies.
+
+If you are an instructor, CCV can provide access to JupyterHub for your class or workshop, and Digital Learning and Design (DLD) can assist with integrating computational assignments into curricula. The implementation is supported by Brown CIS; please follow the link below to request an instance for your class. CIS staff will respond to your request to begin the setup process. We ask that requests for JupyterHub be made at least two months in advance of expected course  deployment.
```
Comment (mcmcgrath13) [inline services/classroom/jupyterhub.md:14]:
```suggestion
If you are an instructor, CCV can provide access to JupyterHub for your class or workshop, and Digital Learning and Design (DLD) can assist with integrating computational assignments into curricula. The implementation is supported by Brown OIT; please follow the link below to request an instance for your class. CIS staff will respond to your request to begin the setup process. We ask that requests for JupyterHub be made at least two months in advance of expected course  deployment. 
```

File: services/classroom/jupyterhub.md [hunk 3]
```diff
@@ -5,7 +5,13 @@ icon: code-greater-than-or-equal
...
+
+On Brown's JupyterHub, each user is provided a persistent working directory and compute resource allocation. This means the environment you are provided is only accessible by yourself and CIS support staff. 
+
+Once connected to JupyterHub servers, users enter an isolated workspace to write and run code. There are no time limit restrictions or specified lockout times, so feel free to use your personal JupyterHub server as needed and adhere to CCV's Computing Policies.
+
+If you are an instructor, CCV can provide access to JupyterHub for your class or workshop, and Digital Learning and Design (DLD) can assist with integrating computational assignments into curricula. The implementation is supported by Brown CIS; please follow the link below to request an instance for your class. CIS staff will respond to your request to begin the setup process. We ask that requests for JupyterHub be made at least two months in advance of expected course  deployment. 
 
 <a href="https://docs.google.com/forms/d/e/1FAIpQLSct9rFCxLhPIezHI-RYRyEuSnvHrPZLMuUSFRTriIyd_3TAfA/viewform?usp=sf_link">Request a Hub</a>
```
Comment (mcmcgrath13) [inline services/classroom/jupyterhub.md:16]: if you want this to be a button I think you can throw `class="button is-primary"` (or whatever color you want) on the `a` tag

File: services/classroom/jupyterhub.md [hunk 4]
```diff
@@ -14,10 +20,10 @@ JupyterHub provides a convenient way to serve Jupyter Notebooks for multiple use
...
+There are other environments that may be great alternatives for your teaching needs. [Google Colaboratory](https://colab.research.google.com/) provides a similar Jupyter-based notebook environment that is free to use, and depending on your needs may require not set up at all. It also provides free access to GPU and TPU programming, which can be great for deep learning courses. Other new and up-coming platforms such as [https://repl.it](https://repl.it "repl.it") are also great alternatives for teaching and learning programming. To-date JupyterHub provides few benefits over these platforms. We list here few of the differentiating features between Google Colab and JupyterHub to help you decide which choice is right for your course:
```
Comment (mcmcgrath13) [inline services/classroom/jupyterhub.md:23]:
```suggestion
There are other environments that may be great alternatives for your teaching needs. [Google Colaboratory](https://colab.research.google.com/) provides a similar Jupyter-based notebook environment that is free to use, and depending on your needs may require not set up at all. It also provides free access to GPU and TPU programming, which can be great for deep learning courses. Other new and up-coming platforms such as [repl.it](https://repl.it) are also great alternatives for teaching and learning programming. To date, JupyterHub provides few benefits over these platforms. We list here a few of the differentiating features between Google Colab and JupyterHub to help you decide which choice is right for your course:
```

File: services/classroom/jupyterhub.md (hunk 3 above)
Comment (paulstey) [reply services/classroom/jupyterhub.md]: Done! Thank you, @mcmcgrath13!!

File: services/classroom/jupyterhub.md [hunk 5]
```diff
@@ -5,19 +5,23 @@ icon: code-greater-than-or-equal
...
+Brown's JupyterHub service  provides a convenient, cloud-hosted way to serve Jupyter Notebooks for multiple users. Notebooks are launched within a pre-configured computing environment; users do not need to install any software packages. This set-up free environment is ideal for courses and workshops where instructors intend for students to begin coding with minimal obstacles. Jupyter’s flexibility allows instructors to pick the preferred language for a particular context, including Python, Julia, R and many more. Regardless of the language chosen, the Jupyter interface remains the same. To learn more about using Jupyter Notebooks for teaching and learning, [please visit the guide.](https://jupyter4edu.github.io/jupyter-edu-book/jupyter.html)
 
-<a href="https://docs.google.com/forms/d/e/1FAIpQLSct9rFCxLhPIezHI-RYRyEuSnvHrPZLMuUSFRTriIyd_3TAfA/viewform?usp=sf_link">Request a Hub</a> 
+On Brown's JupyterHub, each user is provided a persistent working directory and compute resource allocation. This means the environment you are provided is only accessible by yourself and CIS support staff.
```
Comment (mcmcgrath13) [inline services/classroom/jupyterhub.md:10]:
```suggestion
On Brown's JupyterHub, each user is provided a persistent working directory and compute resource allocation. This means the environment you are provided is only accessible by yourself and OIT support staff. 
```

File: services/classroom/jupyterhub.md [hunk 6]
```diff
@@ -5,19 +5,23 @@ icon: code-greater-than-or-equal
...
 
-<a href="https://docs.google.com/forms/d/e/1FAIpQLSct9rFCxLhPIezHI-RYRyEuSnvHrPZLMuUSFRTriIyd_3TAfA/viewform?usp=sf_link">Request a Hub</a> 
+On Brown's JupyterHub, each user is provided a persistent working directory and compute resource allocation. This means the environment you are provided is only accessible by yourself and CIS support staff. 
 
-<a href="https://docs.ccv.brown.edu/jupyterhub/">Documentation</a>
+Once connected to JupyterHub servers, users enter an isolated workspace to write and run code. There are no time limit restrictions or specified lockout times, so feel free to use your personal JupyterHub server as needed and adhere to CCV's Computing Policies.
+
+If you are an instructor, CCV can provide access to JupyterHub for your class or workshop, and Digital Learning and Design (DLD) can assist with integrating computational assignments into curricula. The implementation is supported by Brown OIT; please follow the link below to request an instance for your class. CIS staff will respond to your request to begin the setup process. We ask that requests for JupyterHub be made at least two months in advance of expected course  deployment.
```
Comment (mcmcgrath13) [inline services/classroom/jupyterhub.md:14]:
```suggestion
If you are an instructor, CCV can provide access to JupyterHub for your class or workshop, and Digital Learning and Design (DLD) can assist with integrating computational assignments into curricula. The implementation is supported by Brown OIT; please follow the link below to request an instance for your class. OIT staff will respond to your request to begin the setup process. We ask that requests for JupyterHub be made at least two months in advance of expected course deployment. 
```

**Comment by mirestrepo** [comment]
@paulstey you are so close! 💪

**Comment by paulstey** [comment]
@mirestrepo: I think this might be good-to-go. Could you please take a look?

File: services/classroom/jupyterhub.md [hunk 7]
```diff
@@ -5,19 +5,23 @@ icon: code-greater-than-or-equal
...
+There are other environments that may be great alternatives for your teaching needs. [Google Colaboratory](https://colab.research.google.com/) provides a similar Jupyter-based notebook environment that is free to use, and depending on your needs may require not set up at all. It also provides free access to GPU and TPU programming, which can be great for deep learning courses. Other new and up-coming platforms such as [repl.it](https://repl.it) are also great alternatives for teaching and learning programming. To date, JupyterHub provides few benefits over these platforms. We list here a few of the differentiating features between Google Colab and JupyterHub to help you decide which choice is right for your course:
```
Comment (mcmcgrath13) [inline services/classroom/jupyterhub.md:21]:
```suggestion
There are other environments that may be great alternatives for your teaching needs. [Google Colaboratory](https://colab.research.google.com/) provides a similar Jupyter-based notebook environment that is free to use, and depending on your needs may require no set up at all. It also provides free access to GPU and TPU programming, which can be great for deep learning courses. Other new and up-coming platforms such as [repl.it](https://repl.it) are also great alternatives for teaching and learning programming. To date, JupyterHub provides few benefits over these platforms. We list here a few of the differentiating features between Google Colab and JupyterHub to help you decide which choice is right for your course:
```

**Review by mcmcgrath13 — APPROVED** [review]
just one typo

**Comment by paulstey** [comment]
Thank you, @mcmcgrath13!!!

**Review by mirestrepo — APPROVED** [review]
🚀

**Comment by paulstey** [comment]
Thank you, @mcmcgrath13 and @mirestrepo!!

**Pull request merged**